test/vite-node/test/self-export.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, it } from 'vitest'
+import { execa } from 'execa'
+import { resolve } from 'pathe'
+import ansiEscapes, { HelloWorld } from '../src/self-export'
+
+it('should export self', () => {
+  expect(ansiEscapes.HelloWorld).eq(HelloWorld)
+  expect(Reflect.get(ansiEscapes, 'default').HelloWorld).eq(HelloWorld)
+  expect(HelloWorld).eq(1)
+})
+
+const cliPath = resolve(__dirname, '../../../packages/vite-node/src/cli.ts')
```

Comment:
You should be able to reference `vite-node/dist/cli.js` to call already built package with `node` and not `esno`

Reply:
Then it's probably fine. I will look into vite-node tests separately then.